recipes/rocketpy/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "rocketpyalpha" %}
+{% set version = "0.9.2" %}
+
+package:
+  name: "rocketpy"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 3e5e2e7b59edeb5b03f514b0b6957438b8d12508167d7d35faadda4383ec8c1c
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - matplotlib >=3.0
+    - numpy >=1.0
+    - pip
+    - python
```

Comment:
It looks like this package is not python2 compatible (at least according to the `setup.py`). If that is the case and you are making this `noarch: python`, change `python` to `python  >=3.6` under `host` and `run`.

Reply:
Switched to `python >= 3.6` indeed.